beta/src/components/MDX/ExpandableExample.tsx

```diff
@@ -40,6 +38,8 @@ function ExpandableExample({
         className="list-none p-8"
         tabIndex={-1 /* there's a button instead */}
         onClick={(e) => {
+          // Escape case for the header anchor link
+          if (e.target instanceof SVGElement) return;
```

Comment:
Nit: please use multiline style for if statements

Reply:
Updated